internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -707,7 +707,35 @@ func StartAllServices(seq runtime.Sequence, data interface{}) (runtime.TaskExecu
 		ctx, cancel := context.WithTimeout(ctx, constants.BootTimeout)
 		defer cancel()
 
-		return conditions.WaitForAll(all...).Wait(ctx)
+		aggregateCondition := conditions.WaitForAll(all...)
+
+		errChan := make(chan error)
+
+		go func() {
+			errChan <- aggregateCondition.Wait(ctx)
+		}()
+
+		lastMessage := aggregateCondition.String()
+		logger.Printf("%s", lastMessage)
+
+		ticker := time.NewTicker(time.Second)
+		defer ticker.Stop()
+
+		for {
+			select {
+			case err := <-errChan:
+				return err
+			case <-ticker.C:
+			}
+
+			latestMessage := aggregateCondition.String()
+
+			if latestMessage != lastMessage {
+				logger.Printf("%s", latestMessage)
```

Comment:
as an idea we discussed during the call: should we repeat this message every N seconds (say, 15 seconds), even if the message hasn't changed?

as there are many messages scrolling through dmesg at early boot, it might be easier to spot the right message this way

Reply:
makes sense, that way the user will also see if a service hangs, since it never disappears from the list. 15s sounds good, I'll try that